scss/components/_actor-sheet.scss

```diff
@@ -26,12 +31,13 @@
     flex-direction: column;
     grid-area: sidebar;
     /* background-color: lightgreen; */
-    border-right: thin solid darkgrey;
 }
 
 .actor-sheet .sidebar {
     overflow: hidden scroll;
     padding: 0 2px 2px 8px;
+    background-color: $sheetLighterBackground;
+    height: 100%;
     /* background-color: lavender; */
```

Comment:
We could probably remove this comment as you're actually adding a distinct sidebar `background-color`.